src/main/java/org/folio/circulation/domain/MultipleRecords.java

```diff
@@ -113,4 +113,12 @@ public JsonObject asJson(
   public Integer getTotalRecords() {
     return totalRecords;
   }
+
+  public <R> Result<MultipleRecords<R>> flatMapRecords(Function<T, Result<R>> mapper) {
```

Comment:
This is now somewhat pedantic, please can this method be moved next to the `mapRecords` method

Reply:
I can push a commit now to do this, if it helps?

Reply:
Yes, please.

Reply:
Done